fix(query): vulnerability_alert_score_epss_features

Tohar-orca opened the pull request:
**Proposed Changes**

- Fix a false negative in `terraform/aws/vulnerability_alert_score_epss_features` in cases where a terraform plan json is scanned instead of a .tf file


I submit this contribution under the Apache-2.0 license.

gabriel-cx commented:
hi @Tohar-orca ,

thank you for your contribution!
kindly be aware of this failing unit test:
<img width="490" alt="image" src="https://github.com/Checkmarx/kics/assets/111127232/63779c8b-e913-4131-8f8d-a39fcb23fc5b">

Tohar-orca commented:
Thanks @gabriel-cx I've fixed the unit tests and added a `negative.json` file
It seems the tests are failing on something unrelated though, could you please re-run them?

gabriel-cx commented:
No problem @Tohar-orca !
I will re-run the tests, the e2e tests sometimes fail but the reason is not related to the PR's changes (re-run make them pass), and is the case here! We have a fix for it on the way!
